app/services/platforms/facebook.ts

```diff
@@ -108,15 +109,16 @@ export class FacebookService extends StatefulService<IFacebookServiceState>
     return this.state.activePage.access_token;
   }
 
-  getHeaders(token = this.oauthToken): Headers {
-    const headers = new Headers();
-    headers.append('Content-Type', 'application/json');
-    headers.append('Authorization', `Bearer ${token}`);
-    return headers;
+  getHeaders(req: IPlatformRequest, useToken: boolean | string) {
+    const token = typeof useToken === 'string' ? useToken : this.oauthToken;
```

Comment:
i'm not sure why we have `useToken` as a boolean or string here since it seems like we never actually check the boolean value of it, why can't it just be an optional param as a string or a default argument like before?

Reply:
If you call `platformAuthorizedRequest` method you will get `useToken=true` as an argument. So the default value will be not applied.

Reply:
it will be though since you never check the truthiness of useToken, just if it's a string

given that `useToken === true` then `typeof useToken === 'string'` will be falsey and `token` will be the default value of `this.oauthToken`